src/templates/components/Sidebar/Sidebar.js

```diff
@@ -57,7 +57,7 @@ class Sidebar extends Component {
         {sectionList.map((section, index) => (
           <SectionComponent
             createLink={createLink}
-            isActive={activeSection === section || sectionList.length === 1}
+            isActive={activeSection === section}
```

Comment:
This behavior was intentional ^

Reply:
I think carret (ChevronSvg) gives user a sense of some kind of `dropdown` or `collapse` action if user clicks on it. 
What do you think?

Reply:
Yeah, I suppose that's fair. But I don't think I understand the purpose/benefit of collapsing a single section menu.